README.md

```diff
@@ -1,11 +1,202 @@
-# Void Docs
+# Void Documentation
 
-Welcome to the Void documentation. This repository contains the source data
-behind <https://docs.voidlinux.org/>. Contributing to this repository follows
-the same protocol as the packages tree.
+Welcome to void-docs, the documentation repository for the [Void Linux Project](https://github.com/void-linux). This repository contains the source data behind <https://docs.voidlinux.org/>. Contributing to this repository follows the same protocol as the packages tree.
 
-## Contributing
+## Documentation Types
 
-See the [Submitting
-Changes](https://docs.voidlinux.org/contributing/void-docs/submitting.html) page
-on the docs.
+Before contributing, ensure you understand the intended purpose and guidelines of each type documentation.
+
+### Handbook 
+
+The goal of the Void Handbook is to step through the most critical elements of Void Linux that most every user will want to consider while installing or operating Void. It is terse and directive in tone.
+
+When proposing new content, ensure it meets these criteria or it will be rejected:
+
+* applies to most or all Void users
+* focuses on critical setup and configuration
+* requires few or no dependencies beyond the base-system
+
+If you can't meet those criteria but still have useful information, then Guides (described below) are likely a better consideration.
+
+The source for the [Void Handbook](https://docs.voidlinux.org/) is [void-linux/void-docs](https://github.com/void-linux/void-docs/) on GitHub.
+
+### Guides
+
+The goal of Guides is to cover specific scenarios that may not be of interest to all users. Examples of Guide documents might include, "full disk encryption," "dual-booting with another OS," etc. 
+
+While Guides are curated, they are the least formal documentation in terms of both tone and directiveness. They offer advice more than truth, i.e., the reader must exercise judgment to decide if it is right for a specific instance of Void Linux. For example, "Updating Packages with XBPS" is more of a truth in Void and belongs in the Handbook, whereas "Booting from Mirrored Drives" is a judgment call and belongs in a Guide as not all users will want to do this.
+
+The source for [Void Guides](https://guides.voidlinux.org/) is [void-linux/void-guides](https://github.com/void-linux/void-guides/) on GitHub.
```

Comment:
This part doesn't exist currently, and we are trying to fit the necessary guides inside of `advanced guides`.